src/rancher.js

```diff
@@ -85,8 +68,15 @@ function listStacks( http, stackUrl) {
 	}, error => error );
 }
 
+function listEnvironments(http) {
+	return http.get("/v1/projects").then(res => res.data.map(elm => {
+		elm.listStacks = listStacks.bind( null, http, elm.links.environments, elm.name );
+		elm.listContainers = http.get.bind( null, elm.links.containers );
```

Comment:
You are right! That is dead code that I do not think has ever been used and I forgot to remove it.